src/bp/ui-studio/src/web/views/OneFlow/diagram/nodes/TriggerContents/index.tsx

```diff
@@ -11,14 +11,34 @@ interface Props {
   editNodeItem: (node: BlockModel, index: number) => void
   selectedNodeItem: () => { node: BlockModel; index: number }
   getConditions: () => any
+  currentLang: string
+  defaultLang: string
 }
 
-const TriggerContents: FC<Props> = ({ node, editNodeItem, selectedNodeItem, getConditions }) => {
+const TriggerContents: FC<Props> = ({ node, editNodeItem, selectedNodeItem, getConditions, currentLang }) => {
   const conditionLabels = getConditions().reduce((acc, cond) => ({ ...acc, [cond.id]: cond.label }), {})
   const selectedCondition = selectedNodeItem()
 
+  const checkMissingTranslations = () => {
+    return node.conditions?.some(condition => {
+      switch (condition.id) {
+        case 'user_intent_is':
+          const utterances = condition.params?.utterances || {}
+          const curLangLength = utterances[currentLang] || 0
+
+          return Object.keys(utterances)
+            .filter(l => l !== currentLang)
+            .some(l => utterances[l] > curLangLength)
```

Comment:
I'm not sure we want to have exactly the same number of utterances in different language. Also, with this logic, if you add one more utterance in the other lang versus the default lang, you will see the "missing translation" message on the default language